docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -63,6 +59,81 @@ Configuration
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.lua.v3.Lua>`
 * This filter should be configured with the name *envoy.filters.http.lua*.
 
+A simple example of configuring Lua HTTP filter that contains only :ref:`inline_code 
+<envoy_v3_api_field_extensions.filters.http.lua.v3.Lua.inline_code>` is as follow:
+
+.. code-block:: yaml
+
+  name: envoy.filters.http.lua
+  typed_config:
+    "@type": type.googleapis.com/envoy.extensions.filters.http.lua.v3.lua
+    inline_code: |
+      -- Called on the request path.
+      function envoy_on_request(request_handle)
+        -- Do something.
+      end
+      -- Called on the response path.
+      function envoy_on_response(response_handle)
+        -- Do something.
+      end
+
+By default, Lua script defined in `inline_code` will be treated as a `GLOBAL` script. Envoy will 
+execute it for every HTTP request.
+
+Per-Route Configuration
+-----------------------
+
+The Lua HTTP filter also can be disabled or overridden on a per-route basis by providing a 
+:ref:`LuaPerRoute <envoy_v3_api_msg_extensions.filters.http.lua.v3.LuaPerRoute>` configuration 
+on the virtual host, route, or weighted cluster.
+
+As a concrete example, given the following Lua filter configuration:
+
+.. code-block:: yaml
+
+  name: envoy.filters.http.lua
+  typed_config:
+    "@type": type.googleapis.com/envoy.extensions.filters.http.lua.v3.lua
+    inline_code: |
+      function envoy_on_request(request_handle)
+        -- do something
+      end
+    source_codes:
+      hello.lua:
+        inline_string: |
+          function envoy_on_request(request_handle)
+            request_handle:logInfo("Hello World.")
+          end
+      bye.lua:
+        inline_string: |
+          function envoy_on_response(response_handle)
+            response_handle:logInfo("Bye Bye.")
+          end
+
+The HTTP Lua filter can be disabled on some virtual host, route, or weighted cluster by the 
+LuaPerRoute configuration as follow:
+
+.. code-block:: yaml
+
+  per_filter_config:
+    envoy.filters.http.lua:
+      disabled: true
+
+We can also refer to a Lua script in the filter configuration by specifying a name in LuaPerRoute. 
+The `GLOBAL` Lua script will be overridden by the referenced script:
+
+.. code-block:: yaml
+
+  per_filter_config:
+    envoy.filters.http.lua:
+      name: hello.lua
+
+.. attention::
+
+  The name `GLOBAL` is reserved for :ref:`Lua.inline_code 
+  <envoy_v3_api_field_extensions.filters.http.lua.v3.Lua.inline_code>`, so don't use it otherwise.
```

Comment:
Incomplete sentence? 

Probably: "... <envoy_v3_api_field_extensions.filters.http.lua.v3.Lua.inline_code>`. Therefore, do not use `GLOBAL` as name for ...".